lib/worker.js

```diff
@@ -341,6 +332,38 @@ class AssetComputeWorker {
         }
     }
 
+    async doPostProcess(rendition) {
+        // /!\ if postprocessing fails, rendition will be failed too
+        // at this point, we have the rendition a worker created, available at rendition.path
+        // naming rules are rendition0.extension, rendition1.extension, etc.
+
+        // put postprocessed rendition in a new file
+        const postProcessedRendition = new Rendition(rendition.instructions, rendition.directory, rendition.index, `post-${rendition.name}`);
+
+        console.log(`Post processing rendition ${rendition.name} from batch => ${postProcessedRendition.name}`);
+
+        const postProcessingEngine = this.postProcessingEngine || imgProcessingEngine;
+        await postProcessingEngine.imageProcess(rendition.path, postProcessedRendition.path, rendition.instructions);
+
+        // point to proper rendition once postprocessing done
+        rendition = postProcessedRendition;
+        this.renditions[rendition.index] = postProcessedRendition;
+    }
+
+    async handleBatchPostProcessFailure(postProcessingTimer,err) {
+        console.error(`rendition post-processing for batch failed after ${postProcessingTimer.durationSec()}: ${err.message || err}`);
```

Comment:
If we keep the timer's reference, you can get rid of the timer parameter in the call and use `this.postProcessingTimer` instead.

Reply:
for now have brought back `this.timers.currentPostProcessing` but I think this is not accurate as is. For batch  currentPostProcessing = postProcessing so am leaving a TODO for now. (which I believe is handled in a separate ticket)

Reply:
@jdelbick dealt with the current timers implementation in depth recently. Could you take a look, @jdelbick ?

Reply:
+1 it should be referenced with `this` instead of passing through a variable